core/templates/dev/head/forms/formBuilder.js

```diff
@@ -672,7 +672,13 @@ oppia.factory('rteHelperService', [
           elt.find(
             'img.oppia-noninteractive-' + componentDefn.name
           ).replaceWith(function() {
-            var jQueryElt = $('<' + this.className.split(' ')[0] + '/>');
+            // Look for a class name starting with oppia-noninteractive-*.
+            var tagNameMatch = /(^|\s)(oppia-noninteractive-\w+)/.exec(
```

Comment:
Uppercase chars -- no, not in an html tag or class name, I think. Digits -- possible but unlikely; on balance, I think we might as well include them.